test/e2e/environment/environment.go

```diff
@@ -22,13 +22,11 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+// Please keep the list in alphabetical, case-sensitive
+// (upper before any lower case character) order.
 var (
-	// Please keep the list in alphabetical order.
-
 	// The test does not work in UserNS (for example, `open /proc/sys/kernel/shm_rmid_forced: permission denied`).
 	NotInUserNS = framework.WithEnvironment(framework.ValidEnvironments.Add("NotInUserNS"))
-
-	// Please keep the list in alphabetical order.
```

Comment:
:+1: for this removal. We no longer need the marker to catch "add new entry at end of list" mistakes during a review.

Reply:
cool!